code-challenges.test.js

```diff
@@ -46,6 +70,27 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe("capitalizer", () => {
+    it("takes in an array of words and returns an array with all the words capitalized.", () => {
+        expect(capitalizer(randomNouns1)).toEqual (["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]) 
+        expect(capitalizer(randomNouns2)).toEqual (["Temperature", "Database", "Chopsticks", "Mango"])
+    })
+})
+
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
\ No newline at end of file
+// Pseudo code:
+// input: an array of words with lowercare
+// output: an array of capitalized words
+// create a function called capitalizer that takes in an array 
+// use .map to iterate the array 
+// then use bracket notation to iterate each element and use toUppercase() function to capitalize the first letter/zero index of each word.  
+
+const capitalizer = (array) =>{
+    return array.map((value) => {
+        return value[0].toUpperCase()+ value.substring (1)
+    })
+}
```

Comment:
Bracket notation is not used to iterate. Also, you didn't mention concatenating the rest of the string to the capitalized first letter.